src/dialog/tooltip/tooltip.stories.ts

```diff
@@ -3,7 +3,9 @@ import { action } from "@storybook/addon-actions";
 import { withKnobs, text, select } from "@storybook/addon-knobs/angular";
 
 import { InformationFilledModule } from "@carbon/icons-angular";
-import { DialogModule, PlaceholderModule, DocumentationModule } from "../../";
+import { DialogModule, PlaceholderModule } from "../../";
+import { DocumentationModule } from "../../documentation-component/documentation.module";
```

Comment:
Some of the other stories are also failing because of the the wrong import location for this module